.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: "cargo"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+    day: "monday"
+    time: "05:00"
+    timezone: "UTC"
+  # Because we only do this weekly, allow for many to be created.
+  open-pull-requests-limit: 20
```

Comment:
```yaml
versioning-strategy: increase
```

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#versioning-strategy

I think we want `increase` in this case since it's an app and we want the version to be updated in both Cargo.lock and Cargo.toml

Reply:
oops, github ui didn't show me you left comments!

anyway, `increase` is not supported by cargo according to the docs.